config/installDependencies.sh

```diff
@@ -55,9 +55,10 @@ buildDir=${1:-"build/gaffer-$gafferMilestoneVersion.$gafferMajorVersion.$gafferM
 
 # Get the prebuilt dependencies package and unpack it into the build directory
 
-dependenciesVersion="1.6.0"
-dependenciesVersionSuffix=""
-dependenciesFileName="gafferDependencies-$dependenciesVersion-$platform.tar.gz"
+dependenciesVersion="2.0.0"
+dependenciesVersionSuffix="a5"
+dependenciesPythonVersion="2"
+dependenciesFileName="gafferDependencies-$dependenciesVersion-Python$dependenciesPythonVersion-$platform.tar.gz"
```

Comment:
Do we even need this script any more? We seem to reference it in the build instructions but nowhere else. If we want to keep it, perhaps it should just be a thin wrapper around `.github/workflows/main/installDependencies.py `?

Reply:
I think it's useful to have for someone wanting to build gaffer with the 'right' dependencies for that particular branch. There are a myriad of other ways we could do this though... though this does make it straight forward, given the way we build.